draft-sharabayko-srt.md

```diff
@@ -41,9 +46,123 @@ TODO Introduction
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
-document are to be interpreted as described in BCP 14 {{RFC2119}} {{!RFC8174}}
+document are to be interpreted as described in BCP 14 {{RFC2119}} {{RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
+# Data Packets
+
+The data packet structure is as following.
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|F|                    Packet Sequence Number                   |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+| PP|O|Enc|R|                   Message Number                  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                           Timestamp                           |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                     Destination Socket ID                     |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                              Data                             +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+
+F: 
+: Packet Flag (1 bit). The flag value of data packet must be 0.
+
+Packet Sequence Number:
+: (31 bits).
+
+PP:
+: Position of Packet (2 bits). This field indicates the position of packet.
+The value "10b" means the first packet, "00b" is packets in the middle, "01b" is the last packet.
+If it is a single data packet, the value is "11b".
+
+O:
+: Order (1 bit).  Indicates whether the message should be delivered in order or not.
+This value is not used in Live Transmission Mode({{live-transmission-mode}}).
+
+Enc:
+: Encryption Flag (2 bits). The flag bits indicate whether or not data is encrypted.
+The value "00b" is not encrypted, "01b" means ecnrypted data with even key, 
+"11b" signifies ecrypted data with odd key.
+
+R:
+: Retransmitted Packet (1 bit). This flag bit is clear when a packet is transmitted very first time.
+If a packet is retransmitted, the flag is set as "1".
+
+Message Number:
+: (26 bits).
```

Comment:
A description is required. Unless looking into source codes, I can't find enough information to explain what it is.